src/lib/x509/x509_crl.cpp

```diff
@@ -43,7 +43,7 @@ X509_CRL::X509_CRL(DataSource& src)
    load_data(src);
    }
 
-X509_CRL::X509_CRL(const std::vector<uint8_t>& vec)
+X509_CRL::X509_CRL(std::span<const uint8_t> vec)
    {
    DataSource_Memory src(vec.data(), vec.size());
```

Comment:
```suggestion
   DataSource_Memory src(vec);
```

`DataSource_Memory` already learned to deal with `std::span<>`